app/views/assignments/_assignment_form_options.html.erb

```diff
@@ -35,12 +35,15 @@
   </div>
 </dl>
 
+
+<% if !GitHubClassroom.enterprise_instance? %>
 <dl class="form">
   <dt><label>Add your starter code from GitHub <small>(optional)</small> </label></dt>
   <dd><input class="textfield js-autocomplete-textfield form-control input-contrast input-block" type="text" name="repo_name" value="<%= @assignment.starter_code_repository.try(:full_name) %>" placeholder="<%= @assignment.starter_code_repository.try(:full_name) || 'Search repositories'%>" autocomplete="off" data-autocomplete-search-endpoint="github_repos"></dd>
   <dd class="dd-autocomplete-suggestions"><%= render_autocomplete_suggestions_container %></dd>
   <input class="js-autocomplete-resource-id" type="hidden" name="repo_id" />
 </dl>
+<% end %>
```

Comment:
Why is starter code not an option for enterprise users?

Reply:
See the [conversation from #240](https://github.com/education/classroom/issues/240#issuecomment-274623526). In short, the Enterprise API does not support the Source Import API. I talked with our campus representative and there was no clear ETA on that feature either.

Reply:
That's too bad :/ Oh well, this is fine for now.

Reply:
With repo-templates I think we will be able to reintroduce this. But for now I think this is a good trade off.